### PR TITLE
[Ukrainian] Translate `/resources/faq.md`

### PR DESCRIPTION
**Pull request by LWJerri**

### CODE REVIEW

File: site/docs/uk/resources/faq.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+Вам слід перечитати [розділ про форматування](https://core.telegram.org/bots/api#formatting-options) в документації до Telegram.
+Байтовий зсув, який згадується в повідомленні про помилку, підкаже вам, де саме у вашому рядку є помилка.
+
+::: tip Передача сутностей замість форматування
+Ви можете попередньо розбирати сутності для Telegram, якщо хочете, та вказати `entities` під час надсилання повідомлення.
+Тоді текст вашого повідомлення може бути звичайним рядком.
+Таким чином, вам не доведеться турбуватися про екранування дивних символів.
+Може здатися, що це потребує більше коду, але насправді це набагато надійніше і безпечніше рішення цієї проблеми.
+Найголовніше, що це значно спрощується за допомогою нашого [parse-mode](../plugins/parse-mode.md) плагіна.
+:::
```
Comment (deptyped) [inline site/docs/uk/resources/faq.md:37]:
```suggestion
::: tip Передача сутностей замість форматування
Ви можете попередньо розібрати сутності для Telegram і вказати `entities` при надсиланні повідомлення.
Тоді текст вашого повідомлення може бути звичайним текстом без форматування.
Таким чином, вам не доведеться турбуватися про екранування незвичайних символів.
Може здаватися, що це потребує більше коду, але це набагато надійніше і більш безпечне рішення цієї проблеми.
А головне, що це значно простіше завдяки плагіну [parse-mode](../plugins/parse-mode.md).
:::
```

File: site/docs/uk/resources/faq.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+### 401 Unauthorized
+
+Ваш токен бота неправильний.
+Можливо, ви думаєте, що він правильний.
+Але це не так.
+Поговоріть з [@BotFather](https://t.me/BotFather), щоб дізнатися, який токен у бота.
```
Comment (deptyped) [inline site/docs/uk/resources/faq.md:44]:
```suggestion
### 401 Unauthorized

Ваш токен бота неправильний.
Можливо, ви вважаєте, що він правильний.
Але це не так.
Зверніться до [@BotFather](https://t.me/BotFather), щоб дізнатися токен вашого боту.
```

File: site/docs/uk/resources/faq.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+Коли користувач блокує вашого бота, ви не можете надсилати йому повідомлення або взаємодіяти з ним будь-яким іншим чином, за винятком випадків, коли ваш бот був запрошений до групового чату, учасником якого є цей користувач.
+Telegram робить це, щоб захистити своїх користувачів.
+Ви не можете нічого з цим зробити.
+
+Ви можете:
+
+- Виправити помилку і, наприклад, видалити дані користувача з вашої бази даних.
+- Проігнорувати помилку.
+- Слухати оновлення `my_chat_member` через `bot.on("my_chat_member")`, щоб отримувати сповіщення, коли користувач заблокує вашого бота.
+  Підказка: Порівняйте поля `status` старого і нового учасника чату.
```
Comment (deptyped) [inline site/docs/uk/resources/faq.md:59]:
```suggestion
### 403 Forbidden: bot was blocked by the user

Ймовірно, ви намагалися надіслати повідомлення користувачеві, однак виникла ця помилка.

Коли користувач блокує вашого бота, ви не зможете надсилати йому повідомлення або іншим чином взаємодіяти з ним, за винятком випадків, коли ваш бот був запрошений до групового чату, учасником якого є цей заблокований користувач.
Telegram робить це для захисту своїх користувачів.
Ви не можете нічого з цим зробити.

Але ви можете:

- Обробити цю помилку і, наприклад, видалити дані користувача з вашої бази даних.
- Ігнорувати помилку.
- Обробляти оновлення `my_chat_member` через `bot.on("my_chat_member")`, щоб отримувати сповіщення, коли користувач блокує вашого бота.
  Порада: Порівняйте поля `status` старого і нового учасника чату.
```

File: site/docs/uk/resources/faq.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+### 404 Not found
+
+Якщо це відбувається під час запуску бота, значить, ваш токен бота неправильний.
+Поговоріть з [@BotFather](https://t.me/BotFather), щоб дізнатися, який у вас токен.
+
+Якщо ваш бот працює нормально більшу частину часу, але раптом ви отримуєте помилку 404, значить, ви робите щось дуже дивне.
+Ви можете запитати нас в [груповому чаті](https://t.me/grammyjs) (або в [російськомовному груповому чаті](https://t.me/grammyjs_ru)).
```
Comment (deptyped) [inline site/docs/uk/resources/faq.md:67]:
```suggestion
### 404 Not found

Якщо це відбувається під час запуску бота, це означає, що токен бота неправильний.
Зверніться до [@BotFather](https://t.me/BotFather), щоб дізнатися токен вашого боту.

Якщо ваш бот працює нормально більшість часу, але раптом ви отримуєте помилку 404, це означає, що ви робите дещо незвичайне.
Ви можете задати нам питання в [англомовному](https://t.me/grammyjs) або [російськомовному](https://t.me/grammyjs_ru) групових чатах.
```

File: site/docs/uk/resources/faq.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+### 409 Conflict: terminated by other getUpdates request
+
+Ви випадково запустили бота двічі під час тривалого опитування.
+Ви можете запустити лише один екземпляр бота.
+
+Якщо ви вважаєте, що запустили бота лише один раз, ви можете просто змінити токен бота.
+Це зупинить всі інші екземпляри.
+Зверніться до [@BotFather](https://t.me/BotFather), щоб зробити це.
```
Comment (deptyped) [inline site/docs/uk/resources/faq.md:76]:
```suggestion
### 409 Conflict: terminated by other getUpdates request

Ви помилково запустили бота двічі під час тривалого опитування.
Ви можете запустити лише один екземпляр бота.

Якщо ви вважаєте, що запустили бота лише один раз, ви можете просто змінити токен вашого бота.
Це зупинить всі інші екземпляри бота.
Зверніться до [@BotFather](https://t.me/BotFather), щоб це зробити.
```

**Review by deptyped — CHANGES_REQUESTED** [review]

**Review by deptyped — APPROVED** [review]

File: site/docs/uk/resources/faq.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
+# Часті Питання
+
+Тут зібрано відповіді на найбільш поширені запитання щодо [типових помилок](#чому-я-отримую-цю-помилку) та [проблем, повʼязаних з Deno](#запитання-про-deno).
+
+Якщо у цьому ЧаПи немає відповіді на ваше запитання, також ознайомтеся з [ЧаПи про ботів](https://core.telegram.org/bots/faq), створеним командою Telegram.
```
Comment (Nazar-Ant) [inline site/docs/uk/resources/faq.md:5]: 1. "У цих частих питаннях"
2. "ознайомтеся з частими питаннями, створеними командою ТГ"

```suggestion
Якщо у цих ЧаПи немає відповіді на ваше запитання, ознайомтеся з [ЧаПи про ботів](https://core.telegram.org/bots/faq), створеними командою Telegram.
```

File: site/docs/uk/resources/faq.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+У довідці по API.
+Також вам, напевно, буде корисним ознайомитися з [посібником](../guide).
+
+## У методі відсутній параметр!
+
+Ні, це не так.
+
+1. Переконайтеся, що у вас встановлена остання версія grammY.
+2. Перевірте, чи є параметр опціональним.
+   Якщо це так, то він знаходиться в об'єкті для необов'язкових опцій під назвою `other`.
```
Comment (Nazar-Ant) [inline site/docs/uk/resources/faq.md:18]:
```suggestion
   Якщо це так, то він знаходиться в об'єкті необов'язкових параметрів під назвою `other`.
```

File: site/docs/uk/resources/faq.md [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+   Передайте `{ назва_параметру: значення }` у цьому місці і це спрацює.
```
Comment (Nazar-Ant) [inline site/docs/uk/resources/faq.md:19]:
```suggestion
   Передайте `{ назва_параметру: значення }` у цьому місці, тоді це спрацює.
```

File: site/docs/uk/resources/faq.md [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+
+**Другий:** Ваш бот став дуже популярним і має багато користувачів (сотні тисяч).
+Якщо ви вже переконалися, що використовуєте мінімальну кількість запитів до API для найбільш популярних функцій вашого бота, але все одно стикаєтеся з такими помилками, так званим _flood wait_.
+
+Є кілька речей, які ви можете зробити:
+
+1. Прочитати [статтю](../advanced/flood.md) в документації, щоб отримати базове розуміння ситуації.
+2. Використовувати плагін [`transformer-throttler`](../plugins/transformer-throttler.md).
+3. Використовувати плагін [`auto-retry`](../plugins/auto-retry.md).
+4. Зверніться за допомогою до нас у [груповий чат](https://t.me/grammyjs).
```
Comment (Nazar-Ant) [inline site/docs/uk/resources/faq.md:98]:
```suggestion
4. Звернутися по допомогу до нас у [груповий чат](https://t.me/grammyjs).
```

File: site/docs/uk/resources/faq.md [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+- Простіше і швидше почати роботу.
+- Інструментарій значно кращий.
+- Має вбудовану підтримку виконування TypeScript.
+- Не потрібно підтримувати `package.json` або `node_modules`.
+- Має перевірену стандартну бібліотеку.
+
+> Deno був заснований Райаном Далом (Ryan Dahl) - тією ж людиною, яка заснувала Node.js.
+> Він розповів про свої 10 причин жалітись про Node.js у цьому [відео](https://youtu.be/M3BM9TB-8yA).
+
+grammY розробляється в першу чергу для Deno, але вона так само добре підтримує Node.js.
```
Comment (Nazar-Ant) [inline site/docs/uk/resources/faq.md:125]:
```suggestion
grammY розробляється в першу чергу для Deno, але так само добре підтримує Node.js.
```

File: site/docs/uk/resources/faq.md [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+Кілька важливих причин, чому ми любимо Deno більше, ніж Node.js:
+
+- Простіше і швидше почати роботу.
+- Інструментарій значно кращий.
+- Має вбудовану підтримку виконування TypeScript.
+- Не потрібно підтримувати `package.json` або `node_modules`.
+- Має перевірену стандартну бібліотеку.
+
+> Deno був заснований Райаном Далом (Ryan Dahl) - тією ж людиною, яка заснувала Node.js.
+> Він розповів про свої 10 причин жалітись про Node.js у цьому [відео](https://youtu.be/M3BM9TB-8yA).
```
[the remark this answers is not in the document]
Comment (LWJerri) [reply site/docs/uk/resources/faq.md:123]: That's not exactly what we need. The original video is on behalf of the founder of Node.js, where he talks about 10 reasons and regrets about Node.js and about Deno.

File: site/docs/uk/resources/faq.md [hunk 12]
```diff
@@ -0,0 +1,125 @@
...
+Якщо ви вже переконалися, що використовуєте мінімальну кількість запитів до API для найбільш популярних функцій вашого бота, але все одно стикаєтеся з такими помилками, так званим _flood wait_.
+
+Є кілька речей, які ви можете зробити:
+
+1. Прочитати [статтю](../advanced/flood.md) в документації, щоб отримати базове розуміння ситуації.
+2. Використовувати плагін [`transformer-throttler`](../plugins/transformer-throttler.md).
+3. Використовувати плагін [`auto-retry`](../plugins/auto-retry.md).
+4. Зверніться за допомогою до нас у [груповий чат](https://t.me/grammyjs).
+   Там є досвідчені люди.
+5. Можна попросити Telegram збільшити ліміти, але це дуже малоймовірно, якщо ви не виконали кроки 1-3.
```
[the remark this answers is not in the document]
Comment (deptyped) [reply site/docs/uk/resources/faq.md]: 2-й крок загубився

```suggestion
5. Можна попросити Telegram збільшити ліміти, але це дуже малоймовірно, якщо ви не виконали кроки з 1-го по 3-й.
```

File: site/docs/uk/resources/faq.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (LWJerri) [reply site/docs/uk/resources/faq.md:123]: The original video has Russian subtitles. We can ask the user to include them and that's it.

**Review by Nazar-Ant — APPROVED** [review]

File: site/docs/uk/resources/faq.md [hunk 13]
```diff
@@ -0,0 +1,125 @@
...
+Якщо ви вже переконалися, що використовуєте мінімальну кількість запитів до API для найбільш популярних функцій вашого бота, але все одно стикаєтеся з такими помилками, так званим _flood wait_.
+
+Є кілька речей, які ви можете зробити:
+
+1. Прочитати [статтю](../advanced/flood.md) в документації, щоб отримати базове розуміння ситуації.
+2. Використовувати плагін [`transformer-throttler`](../plugins/transformer-throttler.md).
+3. Використовувати плагін [`auto-retry`](../plugins/auto-retry.md).
+4. Звернутися по допомогу до нас у [груповий чат](https://t.me/grammyjs).
+   Там є досвідчені люди.
+5. Можна попросити Telegram збільшити ліміти, але це дуже малоймовірно, якщо ви не виконали 1-й та 3-й крок.
```
Comment (LWJerri) [inline site/docs/uk/resources/faq.md:100]:
```suggestion
5. Можна попросити Telegram збільшити ліміти, але це дуже малоймовірно, якщо ви не виконали кроки з 1-го по 3-й.
```

**Pull request merged**